### DISCUSSION TITLE
Should I add my own similar project to the readme?

### DISCUSSION THREAD

**Discussion by pawamoy** [orhun/git-cliff#731] [Ideas]
I'm maintaining [git-changelog](https://github.com/pawamoy/git-changelog), which is very similar to git-cliff. It's written in Python and uses the Jinja templating engine. Should I send a PR? 

Also, I should definitely mention git-cliff in my own readme, as I sometimes redirect users to it :blush:

**Comment by orhun** [comment]
Hello! Yes, please send a PR :)

**Reply by pawamoy** [reply]
Done! Also updated my own readme to list git-cliff as an alternative :slightly_smiling_face:

**Reply by orhun** [reply]
Awesome, thanks!
